changelog/1.27.0.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "1.27.0"
+description: "Released on 01/19/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.27.0.
+
+### Features ✨
+
+- web: added ability to disable built-in logins to Coder once OIDC
+  authentication has been configured.
+- web: added tutorial on setting up Git OAuth connection to the onboarding steps
+  displayed to administrators when launching a new Coder deployment.
+- web: switched default extensions marketplace to OpenVSX.
+- web: added ability to generate RSA SSH keys.
+- web: added auto-off settings to the **Create Workspace** and **Edit
+  Workspace** forms’ **Advanced** section.
+- web: added functionality to prune images and log image pull process for EC2
+  providers.
+- infra: added support for Podman as an alternative to sysbox for CVM
+  workspaces.
+- infra: added functionality to log all system-level processes executing in the
+  workspace.
```

Comment:
Since this is a headline feature (as well as OpenVSX and maybe other changes) we should move this to the top of the list

Also we should link to the documentation if possible